app/src/main/res/values-es-rES/strings.xml

```diff
@@ -230,9 +230,9 @@
     <string name="no_albums">No hay Álbumes</string>
     <string name="no_artists">No hay Artistas</string>
     <string name="no_audio_ID">"Primero reproduce una canción, luego intenta de nuevo."</string>
-    <string name="no_equalizer">No se encontró ecualizador</string>
+    <string name="no_equalizer">No se encontró nigún ecualizador</string>
```

Comment:
Hi, here the correct word would be `ningún`

```suggestion
    <string name="no_equalizer">No se encontró ningún ecualizador</string>
```

Reply:
That's right, thanks